src/ngraph/op/fused/lstm_cell.cpp

```diff
@@ -319,11 +312,7 @@ shared_ptr<Node> op::LSTMCell::copy_with_new_args(const NodeVector& new_args) co
                                      new_args.at(3),
                                      new_args.at(4),
                                      get_hidden_size(),
-                                     get_activations(),
-                                     get_activation_alpha(),
-                                     get_activation_beta(),
-                                     get_clip(),
-                                     m_input_forget);
```

Comment:
Are you removed all of these parameters intentional?

Reply:
Forgot to undo :)